lib/pigeon/apns_worker.ex

```diff
@@ -2,95 +2,42 @@ defmodule Pigeon.APNSWorker do
   @moduledoc """
     Handles all APNS request and response parsing over an HTTP2 connection.
   """
-  use GenServer
+  use Pigeon.GenericH2Worker, ping_interval: 600_000
```

Comment:
@rslota any chance you can make this ping_interval configurable? some networks (such as AWS) require activity at a higher frequency so the ping has to be lower